Client/src/Components/Sidebar/index.jsx

```diff
@@ -508,6 +522,44 @@ function Sidebar() {
 			</List>
 			<Divider sx={{ mt: "auto" }} />
 
+			{showStarRepo && (
+				<Box
+					sx={{
+						backgroundColor: "white",
+						padding: theme.spacing(4),
+						borderRadius: theme.shape.borderRadius,
+						position: "relative",
+						height: "100px",
+						display: "flex",
+						flexDirection: "column",
+						justifyContent: "center",
+						margin: theme.spacing(3),
+					}}
+				>
+					<Typography style={{ fontStyle: "bold" , marginTop: "5px",color: "black" , fontSize: "16px"  }} variant="body2" color="black">
+						 Star Checkmate
+					</Typography>
+					<Typography style={{  fontSize: "13px" , marginTop: "10px" , color: "black"  }} variant="body2" color="black">
```

Comment:
Hardcoded values, especially font sizes and colors, are inhernetly not maintainable.

Please refactor to make use of the theme.